third_party/gfootball_engine/src/systems/graphics/rendering/opengl_renderer3d.cpp

```diff
@@ -1756,6 +1782,14 @@ void OpenGLRenderer3D::SetClientTextureUnit(int textureUnit) {
   mapping.glClientActiveTexture(GL_TEXTURE0 + (GLuint)textureUnit);
 }
 
+void OpenGLRenderer3D::SetMaxAnisotropy() {
+  DO_VALIDATION;
+//#ifndef __APPLE__  // Can be used to check for Core Profile Mode on Linux
+  mapping.glTexParameterf(GL_TEXTURE_2D, GL_TEXTURE_MAX_ANISOTROPY_EXT,
+          largest_supported_anisotropy);
+//#endif
+}
+
```

Comment:
I decided to introduce this method to comment out this line on systems where anisotropy filtering is not supported.